plugin/pxr/cmake/defaults/CXXDefaults.cmake

```diff
@@ -17,6 +17,15 @@ include(CXXHelpers)
 include(Version)
 include(Options)
 
+# Require C++11/C++14
+if(${MAYA_APP_VERSION} STRLESS_EQUAL "2019")
+    set(CMAKE_CXX_STANDARD 11)
+elseif(${MAYA_APP_VERSION} STRGREATER_EQUAL "2019")
+    set(CMAKE_CXX_STANDARD 14)
+endif()
```

Comment:
Unfortunately, I think the version of USD factors into this decision as well. USD 20.05 and later will **require** C++14, so for example I think attempting to build maya-usd for Maya 2018 against USD 20.05 would fail with this?